src/dnsmasq_interface.c

```diff
@@ -2171,6 +2184,10 @@ static void FTL_reply(const unsigned int flags, const char *name, const union al
 		//   name = google-public-dns-a.google.com
 		// Hence, isExactMatch is always false
 
+		// Set DNSSEC status to INSECURE if it is still unknown
```

Comment:
Why do you have the exact same code block here and around L2154?

Reply:
This part of the code handles reserve (PTR) lookups, L2154 is the code region where "forward" queries (such as A, AAAA, ...) are handled